test/parallel/test-process-setup-cpu-usage-error.js

```diff
@@ -0,0 +1,16 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+
+const internalProcess = require('internal/process');
+
+const errMsg = 'setup_cpuUsage failed';
+process.cpuUsage = () => { throw new Error(errMsg); };
```

Comment:
This should not throw but rather return something like "test".

Reply:
I'd go with `'"mock_value"'` so the error message will be `'Unable to obtain cpu usage "mock_value"'`